wps_shared/wps_shared/db/models/fuel_type_raster.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import Column, Integer, String
+from wps_shared.db.models import Base
+from wps_shared.db.models.common import TZTimeStamp
+
+
+class FuelTypeRaster(Base):
```

Comment:
What do you think about storing the version in the table as well?

Reply:
Good idea, thanks!